### DISCUSSION TITLE
Unable to configure Parcel (.parcelrc is ignored?)

### DISCUSSION THREAD

**Discussion by mindplay-dk** [codesandbox/codesandbox-client#6674] [Q&A]
Seemingly unable to get the `bundle-text:` import transformer to work, I tried adding a `.parcelrc` to my sandbox:

https://codesandbox.io/s/canopy-9bo3kj?file=/.parcelrc

This is apparently ignored?

I'm guessing CSB is getting it's the Parcel configuration from somewhere else?

I'm sure there's a very specific Parcel configuration required to make this work on the web - and that's probably why `.parcelrc` does not appear on the configuration tab next to `.babelrc` etc.?

Does CSB provide any mean of extending or overriding the built-in `.parcelrc`?

(For the record, what I'm trying to achieve: the `canopy` package has Node.JS dependencies, and I'm trying to make it run in the browser - I've used `alias` in `package.json` to bypass CSB's file system emulation, redirecting it to `fs.js` in the project, where I'm trying to statically import the files required by `canopy` and shimming `readFileSync` to just return the file contents. `bundle-text:` does not seem to work either way - but the reason I need it, is because the `.js` files in `canopy` are not in fact JS files, and the default `"..."` configuration for `bundle-text` would attempt to run the file through standard transforms first; the files need to be pulled in as just text, unmodified, where `canopy` will parse these using a template engine.)

**Comment by JamesACS** [comment]
Hey @mindplay-dk - You're correct, unfortunately it's not possible to edit how Parcel is used to generate Sandboxes. 

For what you're trying to accomplish, I'm afraid I don't have a direct solution but you may be able to find some help in our  [Discord Community](https://discord.gg/vmX36Mvm)
